staging/vhost-device-sound/src/lib.rs

```diff
@@ -105,7 +107,7 @@ pub enum BackendType {
     Alsa,
 }
 
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
```

Comment:
Maybe go all the way since we're changing this line:

```suggestion
#[derive(Debug, Clone, Copy, PartialEq, Eq)]
```